gvm/connections.py

```diff
@@ -220,6 +225,9 @@ def send(self, data):
         else:
             self._stdin.channel.send(data)
 
+    def finish_send(self):
+        return self._stdout.channel.shutdown(1)
```

Comment:
As above, I think using `socketlib.SHUT_WR` or another explanation of the parameter value would make the code even more readable.

Reply:
Oh i wasn't aware of that constant. Thanks for the hint!